crates/dmm-tools/src/dmi.rs

```diff
@@ -159,6 +167,21 @@ impl Image {
         Ok(())
     }
 
+    #[cfg(feature = "png")]
+    pub fn to_raw(&self) -> io::Result<Vec<u8>> {
+        let mut vector = Vec::new();
+        {
+            let mut encoder = png::Encoder::new(&mut vector, self.width, self.height);
```

Comment:
is this safe? It produces an `Encoder<&mut Vec<u8, Global>>`, but I'm wary about the `Write` trait and references to Vec's since there are some circumstances where it'll just blindly write to a vector without updating the size

Reply:
apparently it's fine

![a](https://i.tigercat2000.net/2022/05/WindowsTerminal_zFnBnDcYbw.png)